CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # [Changelog](https://github.com/yola/drf-madprops)
 
+## 0.3.0
+* New version compatible with Django 1.10+
```

Comment:
The big issue her is that it *requires* >= 1.10.5. I'd expect that to be in the changelog